cluster/ephemeral-provider-common.sh

```diff
@@ -2,24 +2,48 @@
 
 set -e
 
-_cli='docker run --privileged --rm -v /var/run/docker.sock:/var/run/docker.sock kubevirtci/cli@sha256:c42004c9626e6a6e723a2107410694cb80864f3456725fdf945b1ca148ed6eaa'
+_prefix=${JOB_NAME:-${PROVIDER}}
+_prefix=${_prefix}${EXECUTOR_NUMBER}
+
+_cli='docker run --privileged --rm -v /var/run/docker.sock:/var/run/docker.sock kubevirtci/cli@sha256:6d7380015a2743992a1b3e61dfe4e79925e825e40a93e4594c125822d36c56fc'
 
 function _main_ip() {
     echo 127.0.0.1
 }
 
+function _port() {
+    ${_cli} port --prefix $_prefix "$@"
+}
+
 function prepare_config() {
     BASE_PATH=${KUBEVIRT_PATH:-$PWD}
     cat >hack/config-provider-$PROVIDER.sh <<EOF
 master_ip=$(_main_ip)
 docker_tag=devel
 kubeconfig=${BASE_PATH}/cluster/$PROVIDER/.kubeconfig
-docker_prefix=localhost:5000/kubevirt
+docker_prefix=localhost:$(_port registry)/kubevirt
 manifest_docker_prefix=registry:5000/kubevirt
 EOF
 }
 
+function _registry_volume() {
+    if [ -n "${JOB_NAME}" ]; then
```

Comment:
There's a lot of branching logic around "JOB_NAME". If JOB_NAME then do this, if not do this. 

I'd rather see a single code path and have JOB_NAME something that is always present regardless of CI or local dev.

Reply:
fixed